spr/src/commands/list.rs

```diff
@@ -58,7 +58,7 @@ fn print_pr_info(
             }
             Some(
                 search_query::PullRequestReviewDecision::CHANGES_REQUESTED,
-            ) => console::style("Rejected").red(),
+            ) => console::style("Changes Needed").red(),
```

Comment:
```suggestion
            ) => console::style("Changes Requested").red(),
```
Maybe we should stick to GitHub's lingo then.

Reply:
Suggestion accepted :)